release-notes/docs/3.11.0.md

```diff
@@ -0,0 +1,201 @@
+# 3.11.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Breaking changes](#breaking-changes)
+- [Supported software](#supported-software)
+- [Highlights](#highlights)
+- [And more...](#and-more)
+
+## Known issues
+
+- [medic-android#127](https://github.com/medic/medic-android/issues/127): Image upload forms crash the app. This has been broken for some time and is not easy to fix while supporting Android 4.4 so the resolution has been deferred until we can make this breaking change. Reach out if you require this feature in the near future.
+
+## Upgrade notes
+
+- This upgrade can be rolled out remotely. Users will download the new version in the background and be prompted to reload the app when it's ready. A small amount of data will be needed to download the new version.
+- Upgrading to this version does not require other applications to be upgraded. This release does not drop support for any hardware or software that works with the previous version.
+- This release modifies some CouchDB view definitions so we recommend you Stage the upgrade and wait for the views to be rebuilt before rolling it out to minimize downtime.
+- It is recommended to test the upgrade on a clone of your server first to ensure it works well for your application.
+
+### Breaking changes
+
+#### Removal of Reporting Rates feature
+
+This feature wasn't documented or tested and hasn't been used in production for some time so it has been removed. To read more about this feature or if this breaking change affects your read this [forum post](https://forum.communityhealthtoolkit.org/t/feature-removal-proposal-reporting-rates/1053).
+
+- [cht-core#6248](https://github.com/medic/cht-core/issues/6248): Remove the Reporting Rates feature
+
+### UI/UX changes
+
+#### Android app
+
+A few UI/UX changes were made to the medic-android app. It is safe to ignore this section if you are not updating android app right now.
+
+##### Add UI for prominent disclosure when requesting for permissions
+
+This was required to comply with the latest Google Play Store requirements.
+
+- [medic-android#136](https://github.com/medic/medic-android/issues/136) 
+
+![prominent disclosure](./images/3.11.0-136.gif)
+
+##### Improve connection errors UX, especially for the XWalk-Webview migration process
+
+If the user does not have a connection the first time the app is opened they will now see a user friendly error message with a retry button.
+
+- [medic-android#163](https://github.com/medic/medic-android/issues/163)
+
+![connection errors](./images/3.11.0-163.gif)
+
+##### Improve UX of crosswalk to webview migration
+
+When migrating data from a crosswalk apk to a webview apk the user is now guided through the migration process so it can be upgraded without support. Read more about the transition to webview in this [previously released issue](https://github.com/medic/medic-android/issues/106).
```

Comment:
I can't find anything in the two links that describes when or why there is a move from crosswalk to webivew. Is there something somewhere that describes that?

Reply:
I've added an extra comment to explain.